ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -623,7 +623,7 @@ function upgrade() {
     set_channel $TARGET_VERSION
     local retry=3 conditional_updates
     while (( retry > 0 )); do
-        conditional_updates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]?.release.version' | xargs)
+        conditional_updates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[] | select (.conditions[].status!="True")| .release.version' | xargs)
```

Comment:
it will throw error when there is not conditonalUpdates in clusterversion, so here we should add `?` back.

Reply:
Thanks @jiajliu, Updated the syntax.